x-pack/plugins/ml/common/types/anomaly_detection_jobs/datafeed.ts

```diff
@@ -30,8 +32,8 @@ export interface ChunkingConfig {
   time_span?: string;
 }
 
-interface Aggregation {
-  buckets: {
+export interface Aggregation {
+  [key: string]: {
```

Comment:
I'm unsure we should do this. AFAIK this will allow for any type of string based key to be valid for this interface like `date_histoasdf`.

Reply:
I think this is a tricky one. Talked to Ben and apparently the users don't necessarily need to name the item as `buckets` for the datafeed config to be valid. Right now on the UI side, we don't support that. I can revert this change back for this PR, but we'll address this issue in a follow up PR.